testdata/config/listcheck.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: listchecker
+metadata:
+  name: staticversion
+  namespace: istio-config-default
+spec:
+  # providerUrl: ordinarily black and white lists are maintained
+  # externally and fetched asynchronously using the providerUrl.
+  overrides: ["v1", "v2"]  # overrides provide a static list
+  blacklist: false
```

Comment:
including this just to have the example + docs ?

Reply:
Yes, this is used in the docs.